net-dialup/freeradius/freeradius-3.0.17.ebuild

```diff
@@ -79,7 +80,8 @@ src_prepare() {
 	use pam || { rm -r src/modules/rlm_pam || die ; }
 	use python || { rm -r src/modules/rlm_python || die ; }
 	use rest || { rm -r src/modules/rlm_rest || die ; }
-	# Do not install ruby rlm module, bug #483108
```

Comment:
Any specific reason to nuke this comment?

Reply:
Still applies.

Reply:
Nope, didn't realize I did.